tests/test_txn.py

```diff
@@ -234,8 +235,11 @@ def test_read_only_txn(self):
         """Tests read-only transactions. Read-only transactions should
         not advance the start ts nor should allow mutations or commits."""
 
-        query = '{ me() {} }'
+        # We sleep here so that rollups do not move the MaxAssigned.
+        # Starting Dgraph v23, rollups can move the MaxAssigned too.
+        time.sleep(1)
```

Comment:
Do we want to add a link to https://github.com/dgraph-io/dgraph/pull/8774?

Reply:
Done.